src/database/models/trip.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+/* eslint-disable no-unused-vars */
+/* eslint-disable strict */
+
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Trips extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+     static associate(models) {
+      Trips.belongsTo(models.Accomodation,{foreignKey:'accomodationId'})
+      models.Accomodation.hasMany(models.Trips,{foreignKey:'accomodationId'});
+      models.Users.hasMany(models.Trips,{
+        foreignKey: {
+          name: 'userId'
+        }
+      });
+    }
+  }
+
+  Trips.init(
+    {
+      id: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+        allowNull: false,
+        primaryKey: true
+      },
+      userId:{
+        type: DataTypes.UUID,
+        allowNull: false,
+      },
+      accomodationId:{
+        type:DataTypes.INTEGER,
+        allowNull: false,
+      },
```

Comment:
Think of adding away for the user to inform the services he will use most to for managers to know if they are available in the time the user wants to use the accommodation services( facilities) . Hint : Implement booking functionality on the services where booking is allowed.